compose/config/config.py

```diff
@@ -134,6 +134,9 @@ def version(self):
             return 1
         return version
 
+    def get_service(self, name):
+        return self.get_service_dicts()[name]
```

Comment:
I think we might want to do either `.get(name)` or a try/except `KeyError` raise a ConfigurationError. It might not be possible for that condition to happen now, but as code changes it would be good to raise an appropriate error.